.vscode/settings.template.json

```diff
@@ -0,0 +1,5 @@
+{
+    "pylint.args": [
+        "--rcfile=clients/python/.pylintrc"
+    ]
+}
```

Comment:
Maybe add an extra empty line here just not to see this annoying sign here in gh